samples/drivers/mbox_data/README.rst

```diff
@@ -0,0 +1,88 @@
+.. zephyr:code-sample:: mbox_data
+   :name: MBOX Data
+   :relevant-api: mbox_interface
+
+   Perform inter-processor mailbox communication using the MBOX API with data.
+
+Overview
+********
+
+This sample demonstrates how to use the :ref:`MBOX API <mbox_api>`.
```

Comment:
probably needs a bit more text here. How does it differ from other available samples / what aspect in particular does it demonstrate?

Reply:
Thanks, I have extended the descriptions that this sample is used to test mbox in data transfer mode for drivers supporting this feature.